### DISCUSSION TITLE
Undo and Redo options in Android cursor menu no longer available on Plain Text

### DISCUSSION THREAD

**Discussion by effieeee** [standardnotes/forum#2759] [General]
Hi, I know it was stated that undo functions were not planned, but up until the latest update on Android I actually was able to use an undo (and redo) button from the cursor menu, but that no longer seems possible with the latest release. I just wanted to know whether this was an intentional change.

_Originally posted by @Hunter-K13 in https://github.com/standardnotes/forum/issues/1412#issuecomment-1284451242_

**Comment by tired-creating-those-usernames** [comment]
4 years and still no basic function as undo.
I can't believe it. 
You paste an important text, then accidentally delete it and voila, you're done, because there is no basic undo.
bravo 👏👏👏
